gettingStartedWithAstraBlock/README.md

```diff
@@ -1,5 +1,5 @@
 # Overview
-Stream real-time, human-readable blockchain data to your app with Astra Block.
+Stream real-time, human-readable blockchain data to your app with Astra Block. Simply request access to the feature within Astra (we'll enable it for your account within 24 hours) and you can instantly create a blockchain database that auto updates as new blocks are mined.
 
 **In this guide, we'll**
 - Introduction to Astra Block
```

Comment:
I realize these were already in place, but the first two bullets don't match the structure from "In this guide, we'll..." Any suggestions?

Reply:
In this guide, we'll:

- Explore how to stream real-time blockchain data into your Astra database
- Practice this new feature with example queries
- Request access to try Astra Block